{
  "dependencies": {
    "python_class": "serializable_elist", 
    "python_module": "g_sorcery.g_collections", 
    "value": {
      "data": [], 
      "separator": "\n\t"
    }
  }, 
  "description": "Hotfix for ope 2.12 + 2.13", 
  "eclasses": [
    "g-sorcery", 
    "gs-pypi"
  ], 
  "homepage": "http://svn.zope.org/Zope/hotfixes/Products.Zope_Hotfix_20111024", 
  "info": {
    "Author:": "Zope Foundation and Contributors", 
    "Categories": {
      "Framework": [
        "Zope2"
      ], 
      "License": [
        "OSI Approved", 
        "Zope Public License"
      ], 
      "Programming Language": [
        "Python"
      ]
    }, 
    "Home Page:": "http://svn.zope.org/Zope/hotfixes/Products.Zope_Hotfix_20111024", 
    "Keywords:": "security hotfix patch", 
    "License:": "ZPL 2.1", 
    "Package Index Owner:": "tseaver"
  }, 
  "license": "Zope Public License", 
  "longdescription": "Hotfix for ope 2.12 + 2.13", 
  "maintainer": [
    {
      "email": "piatlicki@gmail.com", 
      "name": "Jauhien Piatlicki"
    }
  ], 
  "md5": "e2dc441a62af2d79c8aeb46881d15761", 
  "python_compat": "( python{2_6,2_7,3_2,3_3} )", 
  "source_uri": "https://pypi.python.org/packages/source/P/Products.Zope_Hotfix_20111024/Products.Zope_Hotfix_20111024-1.0.tar.gz"
}